source/sass/style.scss

```
@@ -1,4 +1,5 @@
-@import "./viewport.scss";
+@import "./viewports.scss";
```

```suggestion
@import "./viewport.scss";
```
Так как ваш файл называется **viewport.scss**.